cypress.json

```diff
@@ -1,6 +1,7 @@
 {
+  "baseUrl": "http://localhost:3000",
   "fixturesFolder": "tests/fixtures",
   "integrationFolder": "tests/integration",
   "pluginsFile": "tests/plugins/index.js",
-  "supportFile": "tests/support/index.js"
-}
\ No newline at end of file
+  "supportFile": "tests/support/index.ts"
+}
```

Comment:
cannot comment on the video file but looks like you're uploading a video ` cypress/videos/home-page/home-page.spec.ts.mp4`

Reply:
fixed